src/sage_setup/autogen/interpreters/__main__.py

```diff
@@ -1,9 +1,16 @@
-from __future__ import print_function, absolute_import
-
+import argparse
 import os
-from sage.env import SAGE_SRC
 
 from . import rebuild
 
+parser = argparse.ArgumentParser()
+parser.add_argument("output_dir")
+args = parser.parse_args()
+
+output_dir = args.output_dir
+if not output_dir:
+    from sage.env import SAGE_SRC
+    output_dir = os.path.join(SAGE_SRC, "sage", "ext", "interpreters")
```

Comment:
Does anything else invoke this via `__main__`? If not I'd recommend dropping the default and making the output directory required. Otherwise we risk someone complaining about breaking the existing behavior if we later try to remove the dead code.

It'd also be nice to have a short docstring at the top explaining how you'd run this via `python -m ...` and why the variable output directory is useful to the meson build.

Reply:
Good point, done. (The fallback was also not working before, since positional args are always required)